schemas/test/en/test_list_collector_section_summary_items.json

```diff
@@ -0,0 +1,273 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test List Collector Section Summary Items",
+    "theme": "default",
+    "description": "A questionnaire to test list collector section summary items",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "section",
+            "title": "General insurance business",
+            "summary": {
+                "show_on_completion": true,
+                "items": [
+                    {
+                        "type": "List",
+                        "for_list": "companies",
+                        "title": "Companies or UK branches",
+                        "add_link_text": "Add another UK company or branch",
+                        "empty_list_text": "No UK company or branch added"
+                    }
+                ],
+                "related_answers": [
```

Comment:
This needs to be at the list level as related answers belong to a list. So `items.*.related_answers`.

Reply:
Ok, this is nested in the actual "item" now, so for a given list.